### PR TITLE
Add initial description info to LCA bundle

### PR DESCRIPTION
**Pull request by leo8a**
Added some initial description info to improve what it is currently shown from the console catalog hub. This needs to be updated with product final info, but we can adopt this in the meantime.

Before
Screenshot from 2023-11-28 17-04-30

After
Screenshot from 2023-11-28 20-27-30

### CODE REVIEW

File: bundle/manifests/lifecycle-agent.clusterserviceversion.yaml [hunk 1]
```diff
@@ -21,8 +21,16 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    containerImage: quay.io/openshift-kni/lifecycle-agent-operator
+    description: The Lifecycle Agent Operator for OpenShift provides local (on cluster)
+      lifecycle management services for Single Node Openshift (SNO) clusters.
+    features.operators.openshift.io/disconnected: "true"
```
Comment (jc-rh) [inline bundle/manifests/lifecycle-agent.clusterserviceversion.yaml:27]: I am not 100% sure, but I recall we are supposed to explicitly specify all other features as false.

File: bundle/manifests/lifecycle-agent.clusterserviceversion.yaml [hunk 2]
```diff
@@ -21,8 +21,16 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    containerImage: quay.io/openshift-kni/lifecycle-agent-operator
+    description: The Lifecycle Agent Operator for OpenShift provides local (on cluster)
```
[the remark this answers is not in the document]
Comment (leo8a) [reply bundle/manifests/lifecycle-agent.clusterserviceversion.yaml:25]: sure

[the remark this answers is not in the document]
Comment (browsell) [reply bundle/manifests/lifecycle-agent.clusterserviceversion.yaml:25]: Remove operator from the description, should just be lifecycle agent

**Pull request merged**